vm/actor/src/builtin/market/state.rs

```diff
@@ -111,60 +362,51 @@ impl State {
         })
     }
 
-    pub(super) fn maybe_lock_balance<BS: BlockStore>(
+    fn maybe_lock_balance<BS: BlockStore>(
         &mut self,
         store: &BS,
         addr: &Address,
         amount: &TokenAmount,
     ) -> Result<(), ActorError> {
```

Comment:
This fn should now not return an actor error, since it might cause issues with the caller assuming the exit code is handled correctly (which it isn't here)

Reply:
As discussed, this will return an Actor Error due to the usage of `get_locked_balance` and `get_escrow_balance` returning an ActorError.